packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -261,29 +261,22 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
             blob = response.content;
         }
 
+        let documentAttributes: api.IDocumentAttributes = JSON.parse(blob.content);
```

Comment:
It would also be good to wrap the entire decode/patch/encode logic in a conditional to opportunistically skip the whole step if we're in a scenario where we are already in the correct format and we also don't need to patch the branch.

Reply:
This will fail  for base64 encoded strings - JSON.parse will throw.
Parsing is not possible on any data other than attributes blob - we do not know format of the data beyond encoding data.
Name of variable does not represent intention - it's arbitrary blob, not necessarily attribution blob.
+1 to this code being O(1) when pass-through is possible, and no parsing